code/headers/st7735.hpp

```diff
@@ -105,7 +115,87 @@ namespace r2d2::display {
          * @brief inits the display
          *
          */
-        void init();
+        void init() {
```

Comment:
@itzandroidtab @florianhumblot this is from your implementation, i have no idea could you awnser these  questions please?

Reply:
This is something @itzandroidtab will know more about as I haven't worked with this display.

Reply:
The wait's can probably be turned down a bit. It should work with a rtos as it shoulnt matter if the waits become longer

Reply:
If the waits are tuned down a bit it should be fine.

Reply:
@itzandroidtab @florianhumblot can these waits be made shorter, i think in an RTOS it does not matter how long they are realy since they will only get longer and this function is called only upon construction of the screen

Reply:
I suppose with the RTOS it doesn't really matter, it was mostly because of the startup time of the Due being increased a lot, which might cause issues when debugging. I'll resolve it